tools/esdlcmd-xml/esdl2xml.cpp

```diff
@@ -73,6 +74,10 @@ void parseCommandLine(int argc, char* argv[], Esdl2Esxdl * cmd)
             {
                 cmd->setVerbose(true);
             }
+            else if (stricmp(argv[i], "-m")==0 || stricmp(argv[i], "--hidl-mode")==0)
+            {
+                cmd->setHidlMode(true);
```

Comment:
Again, no need for an option when just generating the xml.

Reply:
Kept the option, but now only used to turn it off if necessary.